### ISSUE TITLE
Using multiple dictionaries at once for spellchecking

### ISSUE THREAD

**Issue by lithium-lx** [florisboard/florisboard#2079]
I type both English and German, sometimes in the same sentence/message.

I would love to see the option to use multiple dictionaries at once, maybe
- as a "secondary language**s**" field in the lang subtype editor
or
- by activating/deactivating dictionaries in the "manage dictionaries" and adding a another option under Spell Checker > Configuration >  Language: "use all active dictionaries"

I think this would be easy to implement, if I will find some time I could even do it myself to help you out.

Greetings from Germany!

**Comment by patrickgold** [comment]
That's what I plan with the multilingual typing support. Atm I am locally working on getting a new language provider set up, and I am incorporating the option to use multiple dictionaries for spell check/word suggestions (and thus multiple languages) at once from the start. Later down the line the subtype editor screen will get a new UI and also a secondary language field.

**Comment by WuerfelDev** [comment]
Awesome, would love to see multilingual glide typing, I guess that's part of the multilingual keyboard in the future?
Thank you so much for your work and for this great keyboard :)

**Comment by patrickgold** [comment]
> Awesome, would love to see multilingual glide typing

Yes in the future glide typing will depend on the word suggestions/spelling dictionaries loaded in through the current subtype, thus if your subtype is multilingual glide typing will also be multilingual.
